src/core/core.interaction.js

```diff
@@ -125,17 +125,18 @@ function getDistanceMetricForAxis(axis) {
  * @param {Chart} chart - the chart
  * @param {object} position - the point to be nearest to
  * @param {string} axis - the axis mode. x|y|xy
+ * @param {boolean} [useFinalPosition] - use the elements animation target instead of current position
```

Comment:
nit: elements -> element's (assuming there's only one element and the animation target belongs to that element)